lib/matplotlib/contour.py

```diff
@@ -1681,6 +1684,13 @@ def _initialize_x_y(self, z):
 data : indexable object, optional
     DATA_PARAMETER_PLACEHOLDER
 
+rasterized : bool, optional
+    *Only applies to* `.contourf`.
+    Rasterize the pcolormesh when drawing vector graphics.  This can
```

Comment:
```suggestion
    Rasterize the contour plot when drawing vector graphics.  This can
```
copy/paste error I think